x-pack/plugins/maps/public/embeddable/map_embeddable.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import _ from 'lodash';
+import React from 'react';
+import { Provider } from 'react-redux';
+import { render, unmountComponentAtNode } from 'react-dom';
+import 'mapbox-gl/dist/mapbox-gl.css';
+
+import { Embeddable } from 'ui/embeddable';
+import { I18nContext } from 'ui/i18n';
+
+import { GisMap } from '../components/gis_map';
+import { createMapStore } from '../store/store';
+import { getInitialLayers } from '../angular/get_initial_layers';
+import {
+  setGotoWithCenter,
+  replaceLayerList,
+  setQuery,
+  setRefreshConfig,
+} from '../actions/store_actions';
+import { setReadOnly } from '../store/ui';
+import { getInspectorAdapters } from '../store/non_serializable_instances';
+
+export class MapEmbeddable extends Embeddable {
+
+  constructor({ onEmbeddableStateChanged, savedMap, editUrl, indexPatterns = [] }) {
+    super({ title: savedMap.title, editUrl, indexPatterns });
+
+    this.onEmbeddableStateChanged = onEmbeddableStateChanged;
```

Comment:
unused for now so remove. will need it when we introduce filtering, but not required just yet.